pkg/network/usm/shared_libraries.go

```diff
@@ -282,7 +291,16 @@ func (w *soWatcher) Start() {
 
 				for _, r := range w.rules {
 					if r.re.Match(path) {
-						w.registry.register(root, libPath, lib.Pid, r)
+						if quircksWaitLib(root + libPath) {
```

Comment:
is netty a fork of openssl or gnutls?

Reply:
it's not a fork it's embed in the netty_tcnative.jar / it's built with boringssl by default IIRC